libavutil/hwcontext_dxva2.c

```diff
@@ -316,10 +316,10 @@ static int dxva2_map_frame(AVHWFramesContext *ctx, AVFrame *dst, const AVFrame *
         goto fail;
     }
 
-    for (i = 0; i < nb_planes; i++)
+    for (i = 0; i < 4; i++)
```

Comment:
Why is this change needed ?

Reply:
because according to https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/hwcontext_dxva2.c#L287, dst->format is hwframe when uploading and nb_planes is returned as 0. https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/hwcontext_d3d11va.c#L374 It's also 4 instead of nb_planes in d3d11. 
If this changed accepted, I will update it to delete the nb_planes variable since it is only used here. Thanks.